player/public/js/baseLauncher.js

```diff
@@ -128,8 +130,12 @@ org.ekstep.contentrenderer.baseLauncher = Class.extend({
 		if (this.sleepMode) return
 		if (TelemetryService.instance && TelemetryService.instance.telemetryStartActive()) {
 			var telemetryEndData = {}
+			var endpageSeen = [{ "endpageseen" : this.isEndPageSeen }]
+			var contentProgress = [{ "progress" : this.contentProgress() }]
+			var contentSummary = (_.isFunction(this.contentPlaySummary)) ? this.contentPlaySummary() : ''
```

Comment:
@rahulshukla I think the `contentPlaySummary()` method should define in the base class

Reply:
@gouravmore  What do you mean by base class here and why?